### PR TITLE
Add tfstacks to brew tap

### PR DESCRIPTION
**Pull request by liamcervante**
Add tfstacks to Hashicorp Homebrew tap.

I notice most of the other PRs adding products here also create a file within the `Formula/` directory. We haven't yet released a single version of `tfstacks` so we don't have release URLs or a version number or SHA256 hashes yet. I'm not sure what the right approach here would be? Should we wait until at least one version has released before merging this, or will it just create the target file instead of updating it during the release process? Thanks!

### CODE REVIEW

**Review by radeksimko — COMMENTED** [review]
I admit this may be "too late" kind of a comment (especially in the context of this repository) but I remember when naming things (repositories mainly) related to Terraform, I was explicitly encouraged to use `terraform` as full name, rather than `tf`.

I recall @apparentlymart specifically had some context there.

TL;DR I would have expected this to be called `terraform-stacks` rather than `tfstacks`, in spite of the verbosity.

That said I don't know enough about the project or whether this was already considered, so take this merely as an attempt of "over-communicating context" at the risk of being irrelevant. 😅

**Review by modrake — APPROVED** [review]

**Comment by liamcervante** [comment]
> I was explicitly encouraged to use terraform as full name, rather than tf

Thanks @radeksimko! I'll follow up with Martin to clarify things, we are probably still within the window of renaming things. FWIW, Martin did see the RFC for the tool and comment on that so I think he at least is okay with this. Will get some clarity on this before merging!
